__tests__/database.test.ts

```diff
@@ -88,4 +92,54 @@ describe('Database', () => {
         .then(_ => true)
         .catch(_ => false)
     )
+
+    // news table tests
+    let newsRecordId = 0
+    let testDate = new Date()
+    test('can insert an announcement', () => insertNews(
+        "Test Title",
+        null,
+        "Test Body",
+        testDate,
+        null
+    )
+        .then(newsId => {
+            newsRecordId = newsId
+            expect(newsId).toBeGreaterThan(0)
+        })
+        .catch(_ => false)
+    )
```

Comment:
I learned the hard way, using promises for these tests doesn't work. You should convert them all to just be asynchronous. Just use my other tests as a template.